commands-yml/commands/device/clipboard/set-clipboard.yml

```diff
@@ -8,9 +8,11 @@ description:
 example_usage:
   java:
     |
+      driver.setClipboard('happy testing');
```

Comment:
setClipboard in Java expects bytearray as the first argument and contentType to be set explicitly

Reply:
```
    /**
     * Set the content of device's clipboard.
     *
     * @param contentType one of supported content types.
     * @param base64Content base64-encoded content to be set.
     */
    default void setClipboard(ClipboardContentType contentType, byte[] base64Content) {
```